configs/tracking.conf

```diff
@@ -15,7 +15,7 @@
 --simulator_num_people=200
 --simulator_num_vehicles=10
 ######### Other config #########
---visualize_tracked_obstacles
+# --visualize_tracked_obstacles
```

Comment:
Can you please revert changes to this config file. I think it's best if this config file uses sort by default because it's a well tested tracker that doesn't have many dependencies.